### ISSUE TITLE
【stream bug】When use “ switchVideo ” new m3u8 url，But old m3u8 stream still exists！pls help

### ISSUE THREAD

**Issue by HimiGame** [MoePlayer/DPlayer#586]
new DPlayer use "A" (m3u8) play~ 

when use :
`player.switchVideo({url: B});`

"B" is New m3u8 url~ 

old "A" m3u8 hls stream still exists  . 

screenshot :

<img width="504" alt="bug" src="https://user-images.githubusercontent.com/4198936/66714756-7fe11400-eded-11e9-93c4-d6d3f8b39cfd.png">

ps.
a)  i check html dom, Confirm that there is only one Dplayer  
b) There is only one voice.
c) only have one Dplayer in my page 

How to fix it ~

**Comment by msterzhang** [comment]
Yes, there is indeed this bug, even if you stop the video before switching, it is useless.
